src/index.js

```diff
@@ -10,9 +10,10 @@ const name = "Benchwarmer"
 
 function main() {
   const additions = context.getAllObjects("footpath_addition")
-  const settings = new Settings(additions)
+  const settings = new Settings()
 
   ui.registerMenuItem(name, () => {
+    settings.initializeAvailableAdditions(additions)
```

Comment:
I'd rather just instantiate the `Settings` object here and keep the constructor. We could also move the `const additions` down here.

Reply:
Updated 👍